### ISSUE TITLE
paddlepaddle 能否提供一下l2_loss

### ISSUE THREAD

**Issue by demeiyan** [PaddlePaddle/Paddle#21905]
你好！请问一下，paddlepaddle现在发布的版本里面有l2_loss函数吗？如果没有，有没有推荐的在paddlepaddle里面手写l2_loss的方式？

**Comment by yaoxuefeng6** [comment]
可以试试这么组 l2 loss
l2_loss = fluid.layers.reduce_mean(fluid.layers.square(fluid.layers.elementwise_mul(x, y)))
